mat/symband.go

```diff
@@ -0,0 +1,127 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mat
+
+import (
+	"gonum.org/v1/gonum/blas"
+	"gonum.org/v1/gonum/blas/blas64"
+)
+
+var (
+	symBandDense *SymBandDense
+	_            Matrix           = symBandDense
+	_            Symmetric        = symBandDense
+	_            Banded           = symBandDense
+	_            RawSymBander     = symBandDense
+	_            MutableSymBanded = symBandDense
+)
+
+// SymBandDense represents a symmetric band matrix in dense storage format.
```

Comment:
How so? Let's say we wanted

```
func (sb *SymBandDense) Add(a,b SymBanded){}
```

How would this method signature look otherwise?